debian/links

```diff
@@ -1 +1,3 @@
-/var/lib/chromium-plugins-extra/eos-chrome-plugin-updater/libwidevinecdm.so /usr/lib/chromium-browser/libwidevinecdm.so
\ No newline at end of file
+#!/usr/bin/dh-exec
```

Comment:
I've never seen this before! I trust that it does what it looks like it must do.

Reply:
Yes, this is needed to be able to set different targets for the links, without dh-exec it doesn't work.